### PR TITLE
added test for pug package

### PR DESCRIPTION
**Pull request by KoljaTM**
Tests TODO #92
- added a test for the pug package

### CODE REVIEW

File: __tests__/wares/pug.test.pug [hunk 1]
```diff
@@ -0,0 +1,6 @@
+doctype html
```
Comment (talentlessguy) [inline __tests__/wares/pug.test.pug:1]: for test files there is a folder called "fixtures"

**Review by talentlessguy — CHANGES_REQUESTED** [review]

File: __tests__/wares/pug.test.ts [hunk 2]
```diff
@@ -5,7 +5,9 @@ import { renderFile } from 'pug'
 
 describe('PUG templates tests', () => {
   it('should render pug template', async () => {
-    const { fetch, app } = InitAppAndTest((_req, res) => void res.render('../__tests__/wares/pug.test.pug', null, { renderOptions: { pageTitle: 'Test title', foo: 'bar' } }))
+    const { fetch, app } = InitAppAndTest(
+      (_req, res) => void res.render('../__tests__/fixtures/views/pug.test.pug', null, { renderOptions: { pageTitle: 'Test title', foo: 'bar' } })
```
Comment (talentlessguy) [inline __tests__/wares/pug.test.ts:9]: fixture files don't have to have `.test.` in their filename. It's for tests. for fixtures just use `test.pug` or smth like this

File: __tests__/wares/pug.test.ts [hunk 3]
```diff
@@ -5,7 +5,9 @@ import { renderFile } from 'pug'
 
 describe('PUG templates tests', () => {
```
Comment (talentlessguy) [inline __tests__/wares/pug.test.ts:6]: Pug is a name so it should be capital case

**Review by talentlessguy — CHANGES_REQUESTED** [review]

File: __tests__/wares/pug.test.ts [hunk 4]
```diff
@@ -3,11 +3,9 @@ import { createServer } from 'http'
 import { InitAppAndTest } from '../../test_helpers/initAppAndTest'
 import { renderFile } from 'pug'
```
Comment (talentlessguy) [inline __tests__/wares/pug.test.ts:4]: why do you import `renderFile` here?

Comment (KoljaTM) [reply __tests__/wares/pug.test.ts]: i had been trying out something and forgot to remove the imports. Is there a specific reason to set
   "noUnusedLocals": false
in the tsconfig?

Comment (talentlessguy) [reply __tests__/wares/pug.test.ts]: i think i accidentally copied it from my another project, gonna remove the setting

**Pull request merged**